client/verta/tests/registry/pydantic_models.py

```diff
@@ -0,0 +1,33 @@
+from pydantic import BaseModel
+
+
+class InnerInputClass(BaseModel):
+    h_dict: dict[str, str]
+    i_list: list[str]
+
+    def __init__(self, h, i):
+        super().__init__(
+            h_dict=h,
+            i_list=i,
+        )
```

Comment:
Are these `__init__()`s actually necessary? 🤔 I would've guessed that `pydantic` only requires the above

```python
    h_dict: dict[str, str]
    i_list: list[str]
```